Library/Homebrew/brew.sh

```diff
@@ -114,6 +114,25 @@ else
   : "${HOMEBREW_OS_VERSION:=$(uname -r)}"
   HOMEBREW_OS_USER_AGENT_VERSION="$HOMEBREW_OS_VERSION"
 
+  # Ensure the system Curl is a version that supports modern HTTPS certificates.
+  HOMEBREW_SYSTEM_CURL_VERSION="$($(command -v curl) --version 2>/dev/null | head -n1 | awk '{print $2}')"
+  # shellcheck disable=SC2086,SC2183
```

Comment:
Can you provide a similar comment to the previous usage above this explaining why it's needed?